discovery-provider/src/queries/health_check.py

```diff
@@ -79,6 +82,28 @@ def play_check():
         sign_response=False
     )
 
+# Health check for latest play stored in the db
+@bp.route("/sol_play_check", methods=["GET"])
```

Comment:
move out of bp into api v1 instead + add maxDrift and check between cache and latest db

Reply:
won't fix per offline discussion